noxfile.py

```diff
@@ -9,7 +9,7 @@
 ROOT = pathlib.Path(__file__).parent
 
 
-@nox.session(venv_backend="mamba")
+@nox.session(python="3.11", venv_backend="mamba")
```

Comment:
This is at your discretion, but would it be good to define a global `DEFAULT_PYTHON="3.11"` above, then reference it here and on line 41? I'm thinking this may make it easier to change in the future.

Reply:
I agree. In fact, I've done exactly that locally but hadn't pushed the change to GitHub yet.